Sources/GRPC/HTTP2ToRawGRPCServerCodec.swift

```diff
@@ -111,6 +111,7 @@ internal final class HTTP2ToRawGRPCServerCodec: ChannelDuplexHandler {
         headers: payload.headers,
         eventLoop: context.eventLoop,
         errorDelegate: self.errorDelegate,
+        remoteAddress: context.channel.remoteAddress,
```

Comment:
@Lukasa given this is in `channelRead` is it safe to force unwrap this optional?

Reply:
No, not all channels can answer this question.